stdlib/influxdata/influxdb/influxdb.flux

```diff
@@ -170,7 +170,7 @@ builtin from : (
         ?token: string,
     ) => stream[{B with _measurement: string, _field: string, _time: time, _value: A}]
 
-// to writes data to an InfluxDB Cloud or 2.x bucket.
+// to writes data to an InfluxDB Cloud or 2.x bucket and returns the written data.
```

Comment:
This is confusing (in raw text, at least). Use`to()` or some other notation or formatting?

Reply:
This is a convention of the fluxdoc format. When this gets rendered into actual documentation, it will be formatted as you suggest.